README.md

```diff
@@ -10,7 +14,7 @@ Create beautiful and testable command-line interfaces.
 
 - [Documentation](http://cleo.readthedocs.io)
 - [Issue Tracker](https://github.com/python-poetry/cleo/issues)
-
+- [Discord](https://discord.gg/jHNNrNrhpC)
```

Comment:
I guess the invite itself navigates to the Cleo channel, but we can additionally underline there's a dedicated channel for Cleo as the caution section mentions.

```suggestion
- [Discord (`#cleo`)](https://discord.gg/jHNNrNrhpC)
```